gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -94,13 +94,24 @@ def add_arguments(self, parser):
               'of the the header fields do not match the field values. Note: '
               'setting this flag or `--infer_annotation_types` incurs a '
               'performance penalty of an extra pass over all variants.'))
+    parser.add_argument(
+        '--vcf_parser_type',
```

Comment:
nit: I think just `--vcf_parser` is a better name for this option. We added `type` for the enum to distinguish an actual class from class choices, but in here, it's clear that it's not really an actual class :)